From 89001b914c36c0113e9e07c61a40278932f52a41 Mon Sep 17 00:00:00 2001
From: Hiep Pham <hiep.pham.zy@renesas.com>
Date: Thu, 25 Mar 2021 15:50:37 +0700
Subject: [PATCH 146/151] arm64: dts: renesas: r9a07g044l2-smarc: Enable
 Watchdog in devicetree

This patch enable both 3 channels wdt0, wdt1 and wdt2 of watchdog timer.

Signed-off-by: Hiep Pham <hiep.pham.zy@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts b/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
index ed5ff43..9939755 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
+++ b/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
@@ -661,3 +661,18 @@
 &xinclk {
 	clock-frequency = <24000000>;
 };
+
+&wdt0 {
+	status = "okay";
+	timeout-sec = <60>;
+};
+
+&wdt1 {
+	status = "okay";
+	timeout-sec = <60>;
+};
+
+&wdt2 {
+	status = "okay";
+	timeout-sec = <60>;
+};
-- 
2.7.4

